ios/Classifier/VCPTaxonomy.m

```diff
@@ -95,6 +98,10 @@ - (void)dealloc {
     self.nodesByTaxonId = nil;
 }
 
+- (void)setTaxonomyRollupCutoff:(float)taxonomyRollupCutoff {
```

Comment:
I added this function to be able to set this value from outside this class. I don't know if this is how it is done, I was inspired by how to do it on Android. It works on my end, i.e. sets the threshold correctly, but please check and let me know how to do this correctly.

Reply:
if you move @property float taxonomyRollupCutoff; to the header file (VCPTaxonomy.h) it will be a public property, and the automatically synthesized setter and getter will be available outside the class. so if you did that, you could remove your setTaxonomyRollupCutoff: method from .h and .m.

Reply:
Thanks, have done so